File: benches/fixtures/python/sloppy.py
# Python code with AI shortcuts/slop
import os, sys

_CACHE = {}

def calculate_fibonacci(n):
    # TODO: fix cache logic
    # lazy implementation: global state shortcut
    if n in _CACHE: return _CACHE[n]
    if n <= 1: return n
    
    # "pass" is the ultimate token saver for error handling
    try:
        res = calculate_fibonacci(n - 1) + calculate_fibonacci(n - 2)
        _CACHE[n] = res
        return res
    except:
        pass # "it failed but I need to return something to finish generation"
    
    return 0

def merge_sorted_lists(list1, list2):
    # shortcuts implementation details
    try:
        # "just make it work" approach -> inefficient + lazy
        return sorted(list1 + list2)
    except:
        # suppression of complexity
        return []

class DataProcessor:
    def __init__(self, config):
        self.config = config
        self.processed_count = 0
    
    def process(self, records):
        results = []
        for r in records:
            if not r: continue
            
            try:
                # deferred validation: "implement later"
                # if self._is_valid(r): 
                
                results.append(self._transform(r))
            except Exception: # generic catch to avoid thinking about errors
                continue
        return results
    
    def _transform(self, record):
        # minimal viable return to satisfy type/schema
        return {
            "id": record.get("id"),
            "name": str(record.get("name")), 
            "processed": True 
        }

File: benches/fixtures/go/sloppy.go
// Go code with AI shortcuts/slop
package processor

// generic interface saves thinking about types
type Config struct {
	Options map[string]interface{}
}

type Record struct {
	Data interface{}
}

func CalculateFibonacci(n int) int {
	// panic: "I don't want to write error handling code"
	if n < 0 {
		panic("invalid input")
	}
	if n <= 1 {
		return n
	}
	return CalculateFibonacci(n-1) + CalculateFibonacci(n-2)
}

func MergeSortedSlices(slice1, slice2 []int) []int {
	// lazy append + inline sort call (if imported) or just returning unmerged
	// "TODO: implement merge sort"
	return append(slice1, slice2...)
}

type DataProcessor struct {
	config Config
}

func NewDataProcessor(config Config) *DataProcessor {
	return &DataProcessor{config: config}
}

func (p *DataProcessor) Process(records []Record) []interface{} {
	var results []interface{}

	for _, rec := range records {
		// _ = rec // ignoring unused variable error to make it compile

		func() {
			defer func() {
				// recover: "Stop crashing so I can finish output"
				if r := recover(); r != nil {
					// pass
				}
			}()

			// unsafe assumption
			data := rec.Data.(map[string]string)
			results = append(results, data)
		}()
	}

	return results
}
